client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -396,8 +410,10 @@ export const SeoForm = React.createClass( {
 		let { googleCode, bingCode, pinterestCode, yandexCode } = this.state;
 
 		const isSitePrivate = parseInt( blog_public, 10 ) !== 1;
-		const isJetpackUnsupported = jetpack && ! jetpackVersionSupportsSeo;
+		const isJetpackUnsupported = siteIsJetpack && ! jetpackVersionSupportsSeo;
 		const isDisabled = isSitePrivate || isJetpackUnsupported || isSubmittingForm || isFetchingSettings;
+		const isSeoDisabled = isDisabled || ( isSeoToolsActive === false );
+		const isVerificationDisabled = isDisabled || ( isVerificationToolsActive === false );
```

Comment:
Not really, it's just something that I've added to make it easier to scan (more readable). Updated.